script.speedtester/resources/lib/utils.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
Looks like you have 2 licenses in the add-on but only refer the Apache license on add-on.xml

Reply:
I can relicense, no problem.